### PR TITLE
feat: Itunes container and route setup

### PR DESCRIPTION
**Pull request by bhaveshpawar07**
### Ticket Link

---

### Related Links

---

### Description

---

### Steps to Reproduce / Test

---

---

### Checklist

- [ ] PR description included
- [ ] `yarn test` passes
- [ ] Tests are [changed or added]
- [ ] Relevant documentation is changed or added (and PR referenced)

### GIF's
pr1
---

### Live Link

---

- http://wednesday-react-template.s3-website.ap-south-1.amazonaws.com/<BRANCH_NAME>

### CODE REVIEW

**Review by naeemdadi1 — COMMENTED** [review]
Don't include form container changes

**Review by naeemdadi1 — COMMENTED** [review]
CI should pass. Watch for errors and resolve it.

File: internals/webpack/webpack.config.prod.js [hunk 1]
```diff
@@ -81,7 +82,7 @@ module.exports = require('./webpack.config.base')({
     // Put it in the end to capture all the HtmlWebpackPlugin's
     // assets manipulations and do leak its manipulations to HtmlWebpackPlugin
     new OfflinePlugin({
-      relativePaths: isProd() || isUAT(),
+      relativePaths: false,
```
Comment (naeemdadi1) [inline internals/webpack/webpack.config.prod.js]: Why are we changing this?

File: internals/utils.js [hunk 2]
```diff
@@ -11,7 +11,7 @@ function getBranchName() {
 }
 
 function getDefaultPublicPath() {
-  return isProd() ? '/react-template/' : '/';
+  return '/';
```
Comment (naeemdadi1) [inline internals/utils.js]: it should work without this change

File: app/utils/routeConstants.js [hunk 3]
```diff
@@ -1,10 +1,26 @@
...
     },
     exact: true
+  },
+  form: {
+    route: '/form',
+    props: {
+      maxwidth: 500,
+      padding: 20
+    },
+    exact: true
```
Comment (naeemdadi1) [inline app/utils/routeConstants.js]: You can remove this part

File: app/utils/history.js [hunk 4]
```diff
@@ -13,5 +13,5 @@ export function getBaseName() {
   return '';
 }
 
-const history = createBrowserHistory({ basename: getBaseName() });
+const history = createBrowserHistory();
```
Comment (naeemdadi1) [inline app/utils/history.js]: Why are we changing this?

File: internals/utils.js (hunk 2 above)
Comment (bhaveshpawar07) [reply internals/utils.js]: Fixed the file.

File: internals/webpack/webpack.config.prod.js (hunk 1 above)
Comment (bhaveshpawar07) [reply internals/webpack/webpack.config.prod.js]: Fixed the file.

File: app/utils/routeConstants.js (hunk 3 above)
Comment (bhaveshpawar07) [reply app/utils/routeConstants.js]: It has been removed , along with form component.

File: app/utils/history.js (hunk 4 above)
Comment (bhaveshpawar07) [reply app/utils/history.js]: Fixed the file.

**Review by naeemdadi1 — APPROVED** [review]

**Comment by rohan-saroha** [comment]
LGTM

**Pull request merged**